modules/apps/depot/depot-service/src/main/java/com/liferay/depot/internal/security/permission/contributor/DepotRoleContributor.java

```diff
@@ -46,37 +46,46 @@ public void contribute(RoleCollection roleCollection) {
 			Group group = _groupLocalService.getGroup(
 				roleCollection.getGroupId());
 
-			if (!group.isDepot()) {
+			if (!group.isCMS() && !group.isDepot()) {
 				return;
 			}
 
 			UserBag userBag = roleCollection.getUserBag();
```

Comment:
I think this can be moved inside the ```if (group.isDepot()) {``` condition as it is only used in that scope

Reply:
done